chapters/packages.tex

```diff
@@ -321,3 +321,158 @@ \section{External Resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+This section describes how a tool can present the library in another language.
+Translated Modelica text is provided by external files, so that no modification of the original Modelica text is required.
+\end{nonnormative}
+
+The texts in following Modelica constructs should be translated:
+\begin{itemize}
+\item description strings of component declarations and classes
+\item strings in the following annotations:
+\begin{itemize}
+\item \lstinline!Text.string!, \lstinline!Text.textString!
+\item \lstinline!missingInnerMessage!, \lstinline!obsolete!, \lstinline!unassignedMessage!
+\item \lstinline!Dialog.group!, \lstinline!Dialog.tab!
+\item \lstinline!Dialog.loadSelector.caption!, \lstinline!Dialog.loadSelector.filter!, \lstinline!Dialog.saveSelector.caption!, \lstinline!Dialog.saveSelector.filter!
+\item \lstinline!Documentation.info!, \lstinline!Documentation.revisions!
+\item \lstinline!Figure.title!, \lstinline!Plot.title!, \lstinline!Curve.legend!
+\end{itemize}
+\end{itemize}
+
+Comments (delimited as well as rest-of-line) are not translated.
+Only literal strings and concatenations of literals strings are translated.
+In order to have parameter values as part of the texts the special substitution syntax is preferable (see \cref{text} and \cref{variable-replacements}), and translators need to be aware of these substrings in order to make good translations.
+
+\begin{example}
+Consider:
+\begin{lstlisting}[language=modelica]
+annotation(…, Text(string = "1st Frequency is %f1."), 
+                Text(string = "2nd Frequency "+"is "+String(f2)+".")…);
+\end{lstlisting}
+In this example only the string \lstinline!"1st Frequency is %f1."! can be translated, and no strings from the second line.
```

Comment:
I like the clarity of the example, but not what it states. :)

To me the ability to support use of `sprintf`-like functions is a strong reason to allow sub-expressions such as `"2nd Frequency " = "is"` to be translated.  Good question for today's language group phone meeting!

Reply:
The GitHub suggestion doesn't cover all the lines to which the conversation applies, so I'll resolve and restart.